content/content-tool/tool/src/java/org/sakaiproject/content/tool/ResourcesAction.java

```diff
@@ -4250,6 +4250,41 @@ public int compare(Group g0, Group g1) {
 					// something failed, group filter will be hidden
 				}
 			}
+
+			//SAK-11647 - Group-aware dropboxes
+			try
+			{
+				String currentUser = SessionManager.getCurrentSessionUserId();
+				Site site = SiteService.getSite(ToolManager.getCurrentPlacement().getContext());
```

Comment:
I think `currentSiteId` is already available in this method and can replace `ToolManager.getCurrentPlacement().getContext()`